node/node.go

```diff
@@ -557,12 +624,12 @@ func (n *Node) SendBulkTransactions(txs []*transaction.Transaction) (uint64, err
 	}
 
 	numOfSentTxs := uint64(0)
-	for shardId, txs := range transactionsByShards {
-		err := n.sendBulkTransactionsFromShard(txs, shardId)
+	for shardId, txsForShard := range transactionsByShards {
```

Comment:
forshard is not needed.

Reply:
I changed this in order to remove GoLand's warnings